### ISSUE TITLE
Chrome freeze

### ISSUE THREAD

**Issue by UltraPower31** [YePpHa/YouTubeCenter#1681]
When opening a new YouTube tab while YouTubeCenter is installed, the whole tab freezes for about 8-10 seconds.

**Comment by Yonezpt** [comment]
Could you give more information? The browser version and YTC version you are using are helpful basic information.

**Comment by UltraPower31** [comment]
Yeah sorry,
I'm using Chrome 42.0.2311.90 m with YouTube Center 2.1.7. Adblock Plus caused the same problem for a while and you had to use the EasyList without element hiding rules as workaround. Now they fixed it in the original EasyList as well. I disabled every extension besides YTC to make sure that it's the cause of the problem. I hope that helped. Thanks in advance :)

**Comment by Yonezpt** [comment]
Version 2.1.7 is outdated, install the developer version and see if the issue still persists. You can find it here: https://github.com/YePpHa/YouTubeCenter/wiki/Developer-Version

**Comment by UltraPower31** [comment]
Thanks, but that doesn't solve the problem.

**Comment by Yonezpt** [comment]
If you disable all other addons except YTC, does the problem persist? What about with YTC also disabled?

**Comment by UltraPower31** [comment]
As I already mentioned in my second post, the tab still freezes when everything besides YTC is disabled and the problem is solved if I disable YTC.

**Comment by Yonezpt** [comment]
I'm sorry, I thought I read it carefully, but it seems like I didn't get that part on the first time. 

Have you tried resetting the settings to default?

If that still doesn't help then the only thing left is to wait for the addon author to check on this issue himself.

**Comment by UltraPower31** [comment]
Yes, I removed the addon to install the developer build, so I have the default settings.

**Comment by tomtom9749** [comment]
I believe this is the same as issue #1638  and I am experiencing the same behavior.

chrome:42.0.2311.90
tampermonkey: 3.10.84
yt-center: Dev 501

this is what i have witnessed
![youtube hitching](https://cloud.githubusercontent.com/assets/9009764/7330692/68499ff8-eac1-11e4-89c6-fb9fa98d810c.PNG)

**Comment by AngryLifeender** [comment]
Diabling "Flex width on page" and "Flex width on channel" fixed the problem for me. #1807 
![ss 2015-05-13 at 09 58 19](https://cloud.githubusercontent.com/assets/12432138/7631170/f17d524c-fa3e-11e4-9ac9-9cd7a446fe35.png)
